monitors/index.js

```diff
@@ -119,31 +153,46 @@ async function run(address, shouldPoll, botMonitorObject, walletMonitorObject, p
 const Poll = async function(callback) {
   try {
     if (!process.env.EMP_ADDRESS) {
-      throw new Error(
-        "Bad environment variables! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party."
-      );
+      throw "Bad environment variables! Specify an `EMP_ADDRESS` for the location of the expiring Multi Party.";
     }
 
     if (!process.env.BOT_MONITOR_OBJECT || !process.env.WALLET_MONITOR_OBJECT) {
-      throw new Error("Bad input arg! Specify a `BOT_MONITOR_OBJECT` & `WALLET_MONITOR_OBJECT` to track.");
+      throw "Bad input arg! Specify a `BOT_MONITOR_OBJECT` & `WALLET_MONITOR_OBJECT` to track.";
     }
 
-    const pollingDelay = process.env.POLLING_DELAY ? process.env.POLLING_DELAY : 10_000;
+    const pollingDelay = process.env.POLLING_DELAY ? process.env.POLLING_DELAY : 10000;
```

Comment:
Why remove the `10_000` syntax? Just curious

Reply:
It felt cleaner. Also the underscore does not work with GCP. When I set the delay to `30_000` in GCP it was fed in as `30` and lost the remaining part of the number and so the bots were polling every 30ms! Removing it here makes it consistent with the GCP env config.